Configurateur/front/lib/app_routes.dart

```diff
@@ -47,6 +51,36 @@ class AppRouter {
           child: LandingPage(),
         ),
       ),
+      GoRoute(
+        path: '/company-profil',
+        pageBuilder: (context, state) => const NoTransitionPage(
+          child: CompanyProfilPage(),
+        ),
+      ),
+      GoRoute(
+        path: '/container-profil',
+        builder: (BuildContext context, GoRouterState state) {
+          if (state.extra != null) {
+            final ContainerListData container =
+                state.extra as ContainerListData;
+            return ContainerProfilPage(container: container);
+          } else {
+            return ContainerProfilPage(
+                container: ContainerListData(
+                    id: null,
+                    createdAt: null,
+                    organization: null,
+                    organizationId: null,
+                    containerMapping: null,
+                    price: null,
+                    address: null,
+                    city: null,
+                    design: null,
+                    informations: null,
+                    saveName: null));
```

Comment:
add some , before the ) for the format of the file (will be simpler to read after that